src/pages/projectSettings/ProjectSettingsTabs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect } from "react";
+import styled from "@emotion/styled";
+import { H2, Disclaimer } from "@leafygreen-ui/typography";
+import { Route, useParams } from "react-router-dom";
+import { routes, ProjectSettingsTabRoutes } from "constants/routes";
+import {
+  AccessTab,
+  EventLogTab,
+  GeneralTab,
+  GitHubCommitQueueTab,
+  NotificationsTab,
+  PatchAliasesTab,
+  PeriodicBuildsTab,
+  ProjectTriggersTab,
+  VariablesTab,
+  VirtualWorkstationTab,
+} from "./projectSettingsTabs/index";
```

Comment:
You shouldn't need to import from `/index` directly is the file not being resolved automatically?

Reply:
No, I actually noticed this happens elsewhere in the codebase (e.g. [constants/index](https://github.com/evergreen-ci/spruce/search?q=%22constants+index%22)). Maybe worth looking into separately?